lib/scanner.js

```diff
@@ -30,13 +33,15 @@ function setConfig (width, height, cam) {
 
 function config (_configuration) {
   configuration = _configuration
-  var licenses = configuration.manatee.license
-  manatee.register('qr', licenses.qr.name, licenses.qr.key)
-  manatee.register('pdf417', licenses.pdf417.name, licenses.pdf417.key)
+  var licenses = _.get('scanner.manatee.license', configuration)
+  manatee.register('qr', _.get('qr.name', licenses), _.get('qr.key', licenses))
+  manatee.register('pdf417', _.get('pdf417.name', licenses), _.get('pdf417.key', licenses))
+  camera = require('./camera').config(_configuration)
 }
 
 function cancel () {
   cancelFlag = true
+  camera.closeCamera()
```

Comment:
Need to check if ``camera`` is defined.

Reply:
Once we fix this we can merge into the new branch for testing.

Reply:
done